src/index.html

```diff
@@ -1,13 +1,361 @@
+                >
+              <div class="testimonia__container">
+                <img
+                  src="images/people/3.jpg"
+                  alt="people 1"
+                  class="testimonia__photo"
+                >
+                <div class="testimonia__name">
+                  Max Henry
+                </div>
+                <div class="testimonia__position">
+                  Chief Design Officer
+                </div>
+                <div class="testimonia__position--credo">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us" id="contact_us">
+        <div class="container">
+          <h2 class="page__section--title">
+            Contact us
+          </h2>
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--desktop-1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info--title">
+                  Phone
+                </div>
+                <a href="tel:+1 234 5555-55-55" class="contact-us__info--link">
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info--title">
+                  Email
+                </div>
+                <a href="mailto:hello@miami.com" class="contact-us__info--link">
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--desktop-4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info--title">
+                  Address
+                </div>
+                <a href="https://goo.gl/maps/BxkVvrkYSLE3DMF88" class="contact-us__info--link">
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--desktop-8-12">
+              <form action="">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="form-field contact-us--fild"
+                >
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="form-field contact-us--fild">
+                <textarea
+                    name="message"
+                    placeholder="Message"
+                    class="form-field contact-us--fild contact-us--fild--last form-field--textarea">
+                </textarea>
```

Comment:
```suggestion
                <textarea
                    name="message"
                    placeholder="Message"
                    class="form-field contact-us--fild contact-us--fild--last form-field--textarea"
                   ></textarea>
```
Інсує баг, коли не відображається placeholder для textarea. Це фікситься неочевидним способом - потрібно зробити отаке форматування, як я навів у прикладі - ОДИН відступ вліво для закриваючої дужки і закриваючий тег на тому ж рядку
<img width="512" alt="Снимок экрана 2021-12-10 в 13 58 07" src="https://user-images.githubusercontent.com/66570142/145571173-ce77e36f-a76b-475a-92ae-3854d12b2929.png">